README.md

```diff
@@ -163,10 +163,29 @@ ANALYTICS=YOUR_GOOGLE_TRACKING_ID
 
 
 ## Tokens tutorials
-### Discord Credentials
+### Discord Bot's Token
 <details>
- <summary>Click here to see Discord API Credentials tutorial!</summary>
- Soon!
+ <summary>Click here to see how to get your bot's token!</summary>
+
+1. Go to <a href="https://discordapp.com/developers/applications)">Discord Developer Portal</a>
+2. Your next step is to go over the menu on the left side of the screen and click “Bot”. It’s the icon that looks like a little puzzle piece.
+> Now you want to click the blue “Add Bot” button
+3. Click the “Yes, do it!” button.
+4. You’ll see a green message, “A wild bot has appeared!”,you’ll also see a “Token” and a blue button you can click called “Click to Reveal Token”
+> As soon as you click that button,your token will be revealed,copy it and remember to keep it secure!
+
+</details>
+
+### Discord Bot's Secret
+<details>
+ <summary>Click here to see how to get your bot's client secret!</summary>
+
+1. Go to <a href="https://discordapp.com/developers/applications)">Discord Developer Portal</a>
+2. Your next step is to go over the menu on the left side of the screen and click “OAuth2”. It’s the icon that looks like a wrench.
+3. Now you want to click the blue “Copy” button,which is under the CLIENT SECRET text to copy your bot secret
```

Comment:
3. Find section named "Client Secret",  under the bot secret click "Copy" button
4. Paste client secret to `.env` - `SECRET=CLIENT_SECRET`